src/ol_infrastructure/lib/ol_types.py

```diff
@@ -63,6 +66,24 @@ class Apps(str, Enum):
     xpro = "xpro"
     ecommerce = "unified-ecommerce"
     mit_learn = "mit-learn"
+    open_metadata = " open-metadata"
+
+
+class K8sGlobalLabels(BaseModel):
+    """Base class for Kubernetes resource labels."""
+
+    labels: dict[str, str]
+
+    def __init__(self, stack_info, app: Apps, ou: BusinessUnit, **kwargs):
+        """Add MIT OL standard labels for app and stack info."""
+        super().__init__(**kwargs)
```

Comment:
Rather than making this an init method, it would be better as a `model_validator`. Pydantic models don't like having custom init methods, because they get auto-generated implicitly.